otb-remote/otb-action-remoteAccessDisconnect

```diff
@@ -32,6 +32,9 @@ fi
 
 if [ "$luci_user" ]; then
 	deluser "$luci_user" || true
+	uci delete rpcd.$luci_user || true
+	uci commit rpcd
+	/etc/init.d/rpcd reload
```

Comment:
I don't think you need to reload rpcd, the `reload_config` command will do it for you.